### ISSUE TITLE
Regarding GSSOC project

### ISSUE THREAD

**Issue by Aniket11007** [Umesh-01/Python-Assistant#23]
Hello sir is this project open for GSSOC 22 i would like to contribute in it.

**Comment by Himanshu1808** [comment]
I would like to work on this issue, can you please assign me? I am a GSSOC'22 participant.
Kindly assign me this issue.

**Comment by Umesh-01** [comment]
Yes @Aniket11007 this project is open for GSSoC 2022 contributions. Can you explain what type of contribution you want to make??
